docs/ContribOperators.md

```diff
@@ -131,6 +131,8 @@ This version of the operator has been available since version 1 of the 'com.micr
 <dd>Corresponding past and present are same tensor, its size is (2, batch_size, num_heads, max_sequence_length, head_size)</dd>
 <dt><tt>qkv_hidden_sizes</tt> : list of ints</dt>
 <dd>Hidden dimension of Q, K, V: hidden_size, hidden_size and v_hidden_size</dd>
+<dt><tt>softmax_scale</tt> : float</dt>
```

Comment:
Maybe we can use scale instead of softmax_scale? See below graph:

<img width="118" alt="image" src="https://user-images.githubusercontent.com/30328909/213419918-f13b4b86-e3f2-42f3-86a2-99a2411ceec6.png">

Reply:
sure, changed to scale